src/dropdown.ts

```diff
@@ -136,11 +136,16 @@ export default class Dropdown extends LitElement implements FormControl {
 
   set filterable(isFilterable: boolean) {
     if (this.#isFilterable !== isFilterable && isFilterable && !this.multiple) {
-      if (this.#inputElementRef.value && this.selectedOptions[0]?.label) {
```

Comment:
Same logic. Just a rename and I swapped `this.selectedOptions` for `this.lastSelectOption` to fix some single-select bugs where multiple options have a `selected` attribute.